File: Python/10797.py
import sys

def cntViolate(num, car):
    cnt = 0
    for i in car:
        if num == i: cnt += 1
    return cnt

f = lambda : sys.stdin.readline().rstrip()

num = f()

car = f().split()

print(cntViolate(num, car))

File: Java/_10797.java
package Baekjoon;

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class _10797 {

    public static void main(String[] args){

        try {

            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            String num = br.readLine().trim();
            String[] car = br.readLine().split(" ");

            System.out.println(cntViolate(num, car));

        } catch (Exception e) { e.printStackTrace(); }
    }

    public static int cntViolate(String num, String[] car){
        int cnt = 0;
        for(String i : car) if(i.equals(num)) cnt += 1;
        return cnt;
    }
}
